src/Wellcome.Dds/Wellcome.Dds.AssetDomainRepositories/DigitalObjects/DigitalObjectRepository.cs

```diff
@@ -619,74 +670,96 @@ private List<Image> GetOrphans(Dictionary<string, Image> imagesAlreadyOnDlcs, IE
         /// </summary>
         /// <param name="newDlcsImage"></param>
         /// <param name="existingDlcsImage"></param>
+        /// <param name="reasons"></param>
         /// <returns></returns>
-        private Image? GetPatchImage(Image newDlcsImage, Image existingDlcsImage)
+        private Image? GetPatchImage(Image newDlcsImage, Image existingDlcsImage, out List<string> reasons)
         {
+            reasons = new List<string>();
             Image? patchImage = null;
-            const string patchMessageFormat =
+            const string patchMessageStructuredLoggingFormat =
                 "Patch required for {identifier}. Mismatch for {field} - new: {newValue}, existing: {existingValue}";
+            const string patchMessageUIFormat =
+                "PATCH because mismatch for {0} - new: {1}, existing: {2}";
 
             if (existingDlcsImage.String1 != newDlcsImage.String1)
```

Comment:
Probably not worth changing now but I wonder if all these checks could be done via reflection. akin to Protagonist `ChangeManager` but with a whitelist of props to check.